EventsExpress/ClientApp/src/actions/logout-action.js

```diff
@@ -1,12 +1,15 @@
 import eventHelper from '../components/helpers/eventHelper';
+import { AuthenticationService } from '../services';
 import { reset_hub } from './chat';
 import { resetNotification } from './chats';
-import { updateEventsFilters } from './event-list';
+import { updateEventsFilters } from './event-list-action';
 
 export const SET_LOGOUT = "SET_LOGOUT";
 
+const api_serv = new AuthenticationService();
+
 export default function logout() {
-    revokeToken();
+    api_serv.revokeToken();
```

Comment:
Do we have to do `await` here?

Reply:
Actually, we will not do any things with the response, this is only background work, so we can skip `await` to force accelerate the next invocations